solutions/USACO dec15-feast-gold.mdx

```diff
@@ -47,3 +47,70 @@ int main() {
 </CPPSection>
 
 </LanguageSection>
+
+## Solution 2 - Using BFS
+

```

Comment:
Could you add some more explanation about your solution? Maybe adding some ideas about how to come up with this approach.